.github/workflows/deploy-build-action.yml

```diff
@@ -48,6 +54,9 @@ jobs:
           - platform: win
             arch: x86
             python: 3.12.5
```

Comment:
See above about 3.12.7

Reply:
No, only adding 3.13.0 support in this PR. Upgrading others can happen but in a different PR.